modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -153,6 +153,7 @@ protected Download addTaskDownloadBundle(
 
 		download.dest(destinationDir);
 
+		download.onlyIfNewer(true);
```

Comment:
Sounds good.  So this would work with only URL?  wouldn't have to be SF I guess right?